.env.example

```diff
@@ -16,24 +16,22 @@ DB_PASSWORD=postgres
 # TELESCOPE_ENABLED=true
 # DEBUGBAR_ENABLED=true
 
-# BROADCAST_DRIVER=log
-# CACHE_DRIVER=file
-# QUEUE_CONNECTION=sync
-# SESSION_DRIVER=database
-# SESSION_LIFETIME=120
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+QUEUE_CONNECTION=sync
+SESSION_DRIVER=database
+SESSION_LIFETIME=120
 
-# REDIS_HOST=127.0.0.1
-# REDIS_PASSWORD=null
-# REDIS_PORT=6379
+REDIS_HOST=127.0.0.1
+REDIS_PASSWORD=null
+REDIS_PORT=6379
 
 MAIL_MAILER=smtp
-MAIL_HOST=mailhog
+MAIL_HOST=localhost
 MAIL_PORT=1025
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
 MAIL_ENCRYPTION=null
-MAIL_FROM_ADDRESS=noreply@example.com
-MAIL_FROM_NAME="${APP_NAME}"
 
```

Comment:
Why are you removing these? It's good to keep for testing.

Reply:
these are default value set by the mail.php config. no point in repeating them

Reply:
true, fair point